test/fixtures/groups.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```

Comment:
@nikitr Try not to include auto-generated files. Remove if nothing useful is added